src/CFNetwork/CFHost.cs

```diff
@@ -21,7 +21,11 @@ namespace CFNetwork {
 namespace CoreServices {
 #endif
 
-	// used by CFStream.cs (only?)
+	[Deprecated (PlatformName.WatchOS, 8, 0, message: "Use the new Network framework instead.")]
+	[Deprecated (PlatformName.TvOS, 15, 0, message: "Use the new Network framework instead.")]
+ 	[Deprecated (PlatformName.iOS, 15, 0, message: "Use the new Network framework instead.")]
+ 	[Deprecated (PlatformName.MacCatalyst, 15, 0, message: "Use the new Network framework instead.")]
+ 	[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use the new Network framework instead.")]
```

Comment:
you can remove "new" since it's not new anymore (and that won't surface the test of time)